pysatNASA/instruments/methods/cdaweb.py

```diff
@@ -124,16 +407,16 @@ def download(date_array, tag=None, inst_id=None, supported_tags=None,
     ----------
     date_array : array-like
         Array of datetimes to download data for. Provided by pysat.
-    tag : str or NoneType
-        tag or None (default=None)
-    inst_id : str or NoneType
-        satellite id or None (default=None)
+    tag : str
+        tag or None (default='')
+    inst_id : str
+        satellite id or None (default='')
```

Comment:
Fix descriptions here and everywhere.  Minimal allowable improvement made in one suggestion.